cmd/createtree/main.go

```diff
@@ -72,15 +76,21 @@ func createTree(ctx context.Context) (*trillian.Tree, error) {
 
 	conn, err := grpc.Dial(*adminServerAddr, grpc.WithInsecure())
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to dial %v: %v", *adminServerAddr, err)
 	}
 	defer conn.Close()
 
-	tree, err := trillian.NewTrillianAdminClient(conn).CreateTree(ctx, req)
-	if err != nil {
-		return nil, err
+	for {
+		tree, err := trillian.NewTrillianAdminClient(conn).CreateTree(ctx, req)
```

Comment:
nit: Create the client outside of the loop?

Reply:
Done.